projects/widget-test-app/.env.sample

```diff
@@ -2,7 +2,5 @@ VITE_CLIENT_ID=
 VITE_CLIENT_SECRET=
 VITE_IMPERSONATE_USER_ID=
 VITE_PAYMENT_SESSION_URL_OVERRIDE=
-VITE_ENTERPRISE_CORE_API_BASE_URL=
-VITE_ENTERPRISE_WIDGETS_API_BASE_URL=
-PROXY_TARGET_ENTERPRISE_CORE_API=
-PROXY_TARGET_ENTERPRISE_WIDGETS_API=
+VITE_ENTERPRISE_CORE_API_BASE_URL=https://api.enterprise.sandbox.uphold.com/core
```

Comment:
Partners will be targeting sandbox for testing the widget, so let's have it here as a default